cli/cmd/channel_inspect.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/replicatedhq/replicated/cli/print"
+)
+
+// channelInspectCmd represents the channelInspect command
+var channelInspectCmd = &cobra.Command{
+	Use:   "inspect",
+	Short: "Show full details for a channel",
+	Long:  `replicated channel inspect be52315888f23408e2e4dc9242d4cc2c`,
+}
+
+func init() {
+	channelCmd.AddCommand(channelInspectCmd)
+}
+
+func (r *runners) channelInspect(cmd *cobra.Command, args []string) error {
```

Comment:
im concerned about how parseable this output is. a lot of ppl will use cli for automation